src/delete.ts

```diff
@@ -5,34 +5,39 @@ import * as path from "path";
 import * as core from "@actions/core";
 import * as io from "@actions/io";
 
+import * as input from "./input";
 import * as utils from "./utils";
 
 /**
  * Clean up the conda cache directory
  */
 async function run(): Promise<void> {
   try {
-    const cacheFolder = utils.cacheFolder();
-
-    if (fs.existsSync(cacheFolder) && fs.lstatSync(cacheFolder).isDirectory()) {
+    const inputs = await core.group("Gathering Inputs...", input.parseInputs);
+    let pkgsDirs = utils.parsePkgsDirs(inputs.condaConfig.pkgs_dirs);
+    if (pkgsDirs.length) {
```

Comment:
Can we invert this and exit early instead of deeply nesting everything?

```suggestion
    if (!pkgsDirs.length) return;
```